float64 id
float64 distance
float64 radius

nav_msgs/Path trajectory